src/napari_blob_detection/_widget.py

```diff
@@ -52,7 +57,10 @@ def detect_blobs(
         'face_color': 'red',
         'opacity': 0.5,
         'features': {'sigma': all_sigmas},
-        'size': np.sqrt(2) * np.array(all_sigmas),
+        'size': radius * np.array(all_sigmas),
+        # match the scale of the image and the point
+        # point coordinates are not affected 
+        'scale': image.scale,
```

Comment:
If we're doing `scale`, we should probably do `translate`, `rotate`, `shear`, and `affine` to capture all the public state related to the transforms of a layer.

But we can do that in a follow-up PR if this makes the specific use-case work for now.